example/cumulus-tf/main.tf

```diff
@@ -163,17 +163,19 @@ module "cumulus" {
   private_archive_api_gateway = var.private_archive_api_gateway
   api_gateway_stage           = var.api_gateway_stage
 
-  # Thin Egress App settings
+  # Thin Egress App settings. Uncomment to use TEA.
   # must match stage_name variable for thin-egress-app module
-  tea_api_gateway_stage = local.tea_stage_name
-
-  tea_rest_api_id               = module.thin_egress_app.rest_api.id
-  tea_rest_api_root_resource_id = module.thin_egress_app.rest_api.root_resource_id
-  tea_internal_api_endpoint     = module.thin_egress_app.internal_api_endpoint
-  tea_external_api_endpoint     = module.thin_egress_app.api_endpoint
+  # tea_api_gateway_stage         = local.tea_stage_name
+  # tea_rest_api_id               = module.thin_egress_app.rest_api.id
+  # tea_rest_api_root_resource_id = module.thin_egress_app.rest_api.root_resource_id
+  # tea_internal_api_endpoint     = module.thin_egress_app.internal_api_endpoint
+  # tea_external_api_endpoint     = module.thin_egress_app.api_endpoint
 
   log_destination_arn = var.log_destination_arn
 
+  # Cumulus Distribution settings. Remove/comment if not using Cognito.
+  tea_external_api_endpoint = var.cumulus_distribution_url
```

Comment:
Talked with Lauren a bit about the naming here. It would be a breaking change to rename `tea_external_api_endpoint` but that's something we could consider.

Reply:
What deploys the cumulus distribution? Where does the value of `cumulus_distribution_url` come from?

I ask because I don't see what in this PR is actually deploying the Cognito auth API?

Reply:
I think the value should be the output of the cumulus_distribution module:
`tea_external_api_endpoint = module.cumulus_distribution.api_uri`
cumulus_distribution_url is the input of the example/cumulus-tf/terraform.tfvars, and is optional

Reply:
Ah, good catch. Tested this and it gives us the same URL:

https://github.com/nasa/cumulus/pull/2300/commits/2b07efe5b665ff107490cd5a0419e962e15ceb42